scripts/run_unit_tests

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
Runs all tests that can just stay 100% in python. We should add coverage in here next